be/domain.c

```diff
@@ -359,9 +359,9 @@ static const struct m0_be_0type m0_be_0type_seg = {
 	.b0_fini = be_0type_seg_fini,
 };
 
-static void be_domain_log_cleanup(const char           *stob_domain_location,
-				  struct m0_be_log_cfg *log_cfg,
-				  bool                  create)
+M0_INTERNAL void
```

Comment:
look at my comments below, most likely this part becomes unneeded.

Reply:
[1:05 PM] Anatoliy Bilenko
do you think it's due to the following constants being set inside be_domain_log_cleanup():
[1:05 PM] Anatoliy Bilenko
&log_cfg->lc_store_cfg.lsc_stob_id.si_domain_fid,
log_cfg->lc_store_cfg.lsc_stob_domain_location = location;
log_cfg->lc_store_cfg.lsc_stob_domain_init_cfg = "directio=true";
[1:05 PM] Anatoliy Bilenko
?
[1:05 PM] Venkateswarlu Payidimarry
[3:35 PM] Anatoliy Bilenko
&log_cfg->lc_store_cfg.lsc_stob_id.si_domain_fid,
log_cfg->lc_store_cfg.lsc_stob_domain_location = location; log_cfg->lc_store_cfg.lsc_stob_domain_init_cfg = "directio=true";
yes
[1:06 PM] Venkateswarlu Payidimarry
this changes are not reflecting to main "cfg"
[Anatoliy]
Let's export be_domain_log_cleanup() function then.